kube/services/revproxy/gen3.nginx.conf/gearbox-middleware-service.conf

```diff
@@ -0,0 +1,26 @@
+location /gearbox-match/ {
+  if ($csrf_check !~ ^ok-\S.+$) {
+    return 403 "failed csrf check";
+  }
+
+  proxy_next_upstream off;
+  # Forward the host and set Subdir header so api
+  # knows the original request path for hmac signing
+  proxy_set_header   Host $host;
+  proxy_set_header   Subdir /api;
+  proxy_set_header   Authorization "$access_token";
+  proxy_set_header   X-Forwarded-For "$realip";
+  proxy_set_header   X-UserId "$userid";
+  proxy_set_header   X-SessionId "$session_id";
+  proxy_set_header   X-VisitorId "$visitor_id";
+
+  proxy_connect_timeout 300;
+  proxy_send_timeout 300;
+  proxy_read_timeout 300;
+  send_timeout 300;
+
+  set $proxy_service  "${$gearbox_middleware_release_name}";
+  set $upstream http://${$gearbox_middleware_release_name}-service.$namespace.svc.cluster.local;
+  rewrite ^/analysis/(.*) /$1 break;
+  proxy_pass $upstream;
```

Comment:
For this take inspiration from the gearbox one instead of the ocdcanalysistools: https://github.com/chicagopcdc/cloud-automation/blob/pcdc_dev/kube/services/revproxy/gen3.nginx.conf/gearbox-service.conf